### ISSUE TITLE
Feature Request: Synchronize settings (signatures) across computers

### ISSUE THREAD

**Issue by wizonesolutions** [nylas/N1#787]
I'm planning to use N1 on 2 computers, and from what I can tell, I'll have to configure signatures on both. Any way currently to keep these two in sync? Would this be hard to do as an extension or to implement on the Sync Engine?

**Comment by mbilker** [comment]
This would be a Sync Engine thing. The Account IDs for each email account are not shared between computers. A way to implement this would be to allow something like a Nylas ID to tie multiple computers together, or a plugin that allows some kind of syncing.

**Comment by grinich** [comment]
We have some ideas for how to do this, separate from the sync engine. But it's going to be a while before we ship it.

**Comment by mbilker** [comment]
@grinich Alrighty then. I look forward to this!

**Comment by mikecole79** [comment]
The unprecedented level of awesomeness that would come from being able to sync ALL of your settings, including attached accounts, but just signing in to your Nylas ID is staggering.  I've used this email client for about a half hour now, and it's already by far the best email client I've found on Linux, and it's cleaner than outlook.  Can't wait to see what new features are coming up.  

Any idea on an ETA for some part of the features you mentioned above to start sneaking their way into the system?  I'll be very excited to see them!
